Deployment/Jenkins/agent/EC2-Windows-autoscale.groovy

```diff
@@ -48,6 +48,7 @@ pipeline {
         string(name: 'NpmSetupUri', defaultValue: 'https://rpm.nodesource.com/setup_8.x', description: 'URI of node.js NPM setup')
         string(name: 'PrivateIp', defaultValue: '', description: '(Optional) Set a static, primary private IP. Leave blank to auto-select a free IP')
         string(name: 'PypiIndexUrl', defaultValue: 'https://pypi.org/simple', description: 'URL for pypi')
+        string(name: 'RootVolumeSize', description: 'size of the root volume')
```

Comment:
I'd probably put a `defaultValue`, here &mdash; probably for whatever the smallest EBS size there is in the generic Windows AMIs &mdash; or embed some kind of sizing-guidance in the description. It could be potentially be "ungood" if someone, lacking a guiding value, put a smaller-than-AMI-minimum value into the job-field.

Reply:
Added in commit 0a2ffcd5e3df9ca92240f40bde18eb2d8c4293d2